docker_test.go

```diff
@@ -660,12 +806,118 @@ func TestWebhook(t *testing.T) {
 		},
 		"timestamp": "2020-02-02T00:23:09.544Z"
 	  }`)
-	SendEvent(payload_2) //sending duplicate event to check dedup
+	SendEvent(payload_2, "identify", writeKey) //sending duplicate event to check dedup
+
+	// Sending Batch event
+	payload_Batch := strings.NewReader(`{
+		"batch":
+		[
+			{
+				"userId": "identified_user_id",
+				"anonymousId": "anonymousId_1",
+				"type": "identify",
+				"context":
+				{
+					"traits":
+					{
+						"trait1": "new-val"
+					},
+					"ip": "14.5.67.21",
+					"library":
+					{
+						"name": "http"
+					}
+				},
+				"timestamp": "2020-02-02T00:23:09.544Z"
+			}
+		]
+	}`)
+	SendEvent(payload_Batch, "batch", writeKey)
```

Comment:
Should send event accept a reader or a string, if most of the cases we are using `strings.NewReader` before calling it

Reply:
Not sure if this is possible as we need JSON payload and call type both.
Will think of approach where we can pick the JSON payload for external file in testdata folder on the bases of call type.